README.md

```diff
@@ -1 +1,50 @@
-# angular-what-is-angular-readme
+# What is Angular and why does it exist?
+
+## Overview
+
+In this lesson we're going to be looking into AngularJS (commonly referred to as just "Angular"), an open-source framework that was released in 2009. It is currently being mainly maintained by Google as well as a massive community of developers.
+
+It aims to solve the problems faced when creating single page applications, implementing a MVVM pattern to separate presentation and business logic.
+
+## Objectives
+
+- Describe what Angular's role in the web platform is
+- Describe the issues Angular looks to solve
+
+## Frontend Frameworks
+
+Ever wondered how Facebook can have a different timeline for millions of users? How Twitter can display tweets for 300 million people? That's where frontend frameworks come in. 
+
+Frontend frameworks have grown massively in the last 3 years. They help developers structure their applications and create interactive, super quick applications.
+
+### Life before frameworks
+
+Before we had the pleasure of using frontend frameworks, code would be written very procedurally. JavaScript code would become unwieldy as there was no easy way to structure code for each page that the client would see.
```

Comment:
This section makes it seem like front-end frameworks just give us a structure for our app, as in "this code should go here, this should go there, etc."  What else do they give us? For example, with Rails, we talk about how Rails takes away the code that's common to every web application and let's us focus on what's unique to our application.